dev-requirements.txt

```diff
@@ -0,0 +1,18 @@
+annotated-types==0.7.0
+black==25.1.0
+certifi==2025.1.31
+charset-normalizer==3.4.1
+click==8.1.8
+idna==3.10
+iniconfig==2.0.0
+mypy-extensions==1.0.0
+packaging==24.2
+pathspec==0.12.1
+platformdirs==4.3.6
+pluggy==1.5.0
+pydantic==2.10.6
+pydantic_core==2.27.2
+pytest==8.3.4
+requests==2.32.3
+typing_extensions==4.12.2
+urllib3==2.3.0
```

Comment:
The `dev-requirements.txt` file is missing a newline at the end of the file, which can cause issues with some tools that expect a newline at the end of files. Add a newline at the end of the file to resolve this.